--- a/vendor/magento/module-quote/Model/Cart/ShippingMethod.php	2020-08-01 19:48:13.000000000 +0700
+++ b/vendor/magento/module-quote/Model/Cart/ShippingMethod.php.new	2020-08-13 13:17:19.000000000 +0700
@@ -17,12 +17,12 @@
 {
 
     /**
-    * Returns Get Flag.
+    * Returns Get Flag Shipping.
     *
-    * @return string Get Flag.
+    * @return string Get Flag Shipping.
     */
-    public function getFlag(){
-        return $this->_get(self::KEY_FLAG);
+    public function getFlagShipping(){
+        return $this->_get(self::KEY_FLAG_SHIPPING);
     }
 
     /**
@@ -30,8 +30,8 @@
     *
     * @return string Set Flag.
     */
-    public function setFlag($flag){
-        return $this->setData(self::KEY_FLAG, $flag);
+    public function setFlagShipping($flagshipping){
+        return $this->setData(self::KEY_FLAG_SHIPPING, $flagshipping);
     }
 
     /**
